doc/installation_linux_manual.md

```diff
@@ -0,0 +1,169 @@
+<a name="readme-top"></a>
+
+# Linux Ubuntu Installation
+
+<!-- TABLE OF CONTENTS -->
+<details>
+  <summary>Table of Contents</summary>
```

Comment:
Let's align TOC to ROS (meaning not collapsed)
![image](https://github.com/IntelRealSense/librealsense/assets/64067618/bc2d8f2d-4b5c-412b-83cf-c75487bbb3ee)

Reply:
Done